### ISSUE TITLE
Test create activity endpoint prior to merging to dev

### ISSUE THREAD

**Issue by micheal-w-wells** [bcgov/lucy-web#863]
**Describe the Task**
There needs to be a pipeline step to both indicate that the activity endpoint doesn't work with the test API call we have for reference, as well as block merges to dev if it is just broken in general.

@rstens there are some sample json files Jamie made in `/api-mobile/test`, but I am going to update them with the payload from the iOS device (its ordered alphabetical) just for ease of troubleshooting.  Once that is done it would be awesome if you could get your postman newman stuff going and put a pipeline step in to block merges to dev if the endpoint has been broken or the test hasn't been updated.

**Comment by micheal-w-wells** [comment]
Still really relevant.  Leaving as sprint 7 and will carry forward tomorrow if its still here (all good either way, i know you have a new task).

**Comment by rstens** [comment]
Stuck, waiting for Jamie's updates

**Comment by micheal-w-wells** [comment]
Does it just fail on insert?  I think @NickPhura had the idea we could make the spatial inputs/columns optional while we get things going.  I can help you with that if he doesn't already have it in a branch somewhere.  Gonna do some MyRange testing and shipping tomorrow otherwise.

**Comment by rstens** [comment]
Version is working, since the I API does not run yet, I am currently hard pointing this to the https://api-dev-invasivesbc.pathfinder.gov.bc.ca/, once the api runs in its right spot, I will need to update the script/location etc.
